src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+document.querySelector('DOMContentLoaded', () => {
+  const thumbsContainer = document.getElementById('thumbs');
+  const largeImg = document.getElementById('largeImg');
+
+  thumbsContainer.addEventListener('click', function (e) {
+    const target = e.target;
+
+    if (target && (target.tegName === 'IMG' || target.tagName === 'A')) {
+      e.preventDefault();
+
+      let newSrc;
+
+      if (target.tagName === 'A') {
+        newSrc = target.parentElement.href;
```

Comment:
Using 'target.parentElement.href' assumes that the 'A' tag is the direct parent of the clicked element. This might not always be the case. Consider using 'target.href' directly if 'target' is an 'A' tag.